### DISCUSSION TITLE
blog/2024/vlans-on-sr-linux-and-aristacisco/

### DISCUSSION THREAD

**Discussion by giscus** [srl-labs/learn-srlinux#147] [comments]
# blog/2024/vlans-on-sr-linux-and-aristacisco/

Open documentation for open Network OS

https://learn.srlinux.dev/blog/2024/vlans-on-sr-linux-and-aristacisco/

**Comment by jbemmel** [comment]
Would it be possible to interconnect multiple different VLANs on EOS into a single broadcast domain using VLAN translation? https://forums.servethehome.com/index.php?threads/port-specific-tags-in-arista.37773/

```
core-40g(config-if-Et5/1)#switchport vlan translation ?
  $         list end
  <1-4094>  VLAN ID to map from
  in        Map ingress packets only
  out       Map egress packets only
```

**Reply by hellt** [reply]
no
